openfe/utils/system_probe.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import socket
+import subprocess
+
+
+def _get_hostname() -> str:
```

Comment:
not sure this one-liner needs its own function

Reply:
Mostly did this so if we decided to hash it, or later change it to `socket.getfqdn` we could more easily